_pages/about-us/offices/washington-dc.md

```diff
@@ -47,29 +48,27 @@ title: Washington, D.C.
 
 ### From the airport
 
-**Reagan Washington National Airport (DCA)** is the closest airport to D.C. (and GSA). If you&rsquo;re coming from DCA, you can take a cab (~$20) or ride the Metro, which is likely faster. Take the Blue line from the Ronald Reagan Washington National Airport station to Farragut West station. It should take approximately 15 minutes and cost ~$2.15. Once outside the Metro station, follow the directions under "Public Transit."
+**Reagan Washington National Airport (DCA)** is the closest airport to D.C. (and GSA). If you're coming from DCA, you can take a cab (~$20) or ride the Metro, which is likely faster. Take the Blue line from the Ronald Reagan Washington National Airport station to Farragut West station. It should take approximately 15 minutes and cost ~$2.15. Once outside the Metro station, follow the directions under "Public Transit."
 
-If you&rsquo;re coming from **Dulles International Airport (IAD)**, you can take a cab (~$80) or ride public transit, which will take about an hour and cost between $7.00 and $8.25. You can take the 5A bus from Dulles Main Terminal to the Rosslyn station on the Blue/Orange/Silver Line. You can then hop on a Metro to the Farragut West station. More information is available on the Metro site. Once outside the station, follow the directions under "Public Transit."
+If you're coming from **Dulles International Airport (IAD)**, you can take a cab (~$80) or ride public transit, which will take about an hour and cost between $7.00 and \$8.25. You can take the 5A bus from Dulles Main Terminal to the Rosslyn station on the Blue/Orange/Silver Line. You can then hop on a Metro to the Farragut West station. More information is available on the Metro site. Once outside the station, follow the directions under "Public Transit."
```

Comment:
I have my editor configured to format with [Prettier](https://prettier.io) by default, and apparently it escapes dollar signs in case MathJax is being used.

https://github.com/prettier/prettier/issues/5425#issuecomment-437383807

Reply:
Undid these.